fdbclient/include/fdbclient/Tenant.h

```diff
@@ -67,6 +67,23 @@ enum class TenantState { REGISTERING, READY, REMOVING, UPDATING_CONFIGURATION, R
 // Can be used in conjunction with the other tenant states above.
 enum class TenantLockState : uint8_t { UNLOCKED, READ_ONLY, LOCKED };
 
+// metadata used in proxies and stored in the txnStateStore
+struct TenantMinimalMetadata {
+	constexpr static FileIdentifier file_identifier = 265557;
+
+	int64_t id = -1;
+	TenantName tenantName;
```

Comment:
Right now I think we rely on this name for doing CP tenant lookups. Is there anything else that you know of that uses it? I'm wondering if we can remove it once all the various changes are done.

Reply:
No, I can't think of anything. I hope we can remove it.